src/condor_starter.V6.1/starter.cpp

```diff
@@ -3218,25 +3218,16 @@ Starter::transferOutput( void )
 			}
 		}
 
-		jic->transferOutputMopUp();
-
-			/*
-			  there was an error with the JIC in this step.  at this
-			  point, the only possible reason is if we're talking to a
-			  shadow and file transfer failed to send back the files.
-			  in this case, just return to DaemonCore and wait for
-			  other events (like the shadow reconnecting or the startd
-			  deciding the job lease expired and killing us)
-			*/
-		dprintf( D_ALWAYS, "JIC::transferOutput() failed, waiting for job "
-				 "lease to expire or for a reconnect attempt\n" );
-		return false;
-	}
+		// If we've lost the syscall socket at this point, there's no point
+		// in doing the output transfer mop-up, but the shadow might reconnect.
+		if( /* FIXME */ false ) {
```

Comment:
It looks like the case where the shadow is disconnected before the transfer begins is already handled in JICShadow::transferOutput(). It has an early out that sets transient_failure=true. The code just above here then returns without doing mop-up or clean-up. So this FIXME code is useful only for the case where we get disconnected during the output transfer. I think that's a would-be-nice option that can be left for the future.